types/jest/index.d.ts

```diff
@@ -233,15 +254,63 @@ declare namespace jest {
          * Only runs this test in the current file.
          */
         only: It;
+        /**
+         * Skips running this test in the current file.
+         */
         skip: It;
+        /**
+         * Experimental and should be avoided.
+         */
```

Comment:
This is purposefully not documented on the website, but I don't want to break anyone's builds.